orc/src/test/java/org/apache/iceberg/orc/TestTableProperties.java

```diff
@@ -82,19 +93,29 @@ public void testOrcTableProperties() throws Exception {
         DynFields.builder().hiddenImpl(writer.getClass(), "conf").build(writer);
 
     Configuration configuration = confField.get();
-    Assert.assertEquals(String.valueOf(blockSizeBytes), configuration.get(OrcConf.BLOCK_SIZE.getAttribute()));
-    Assert.assertEquals(String.valueOf(stripeSizeBytes), configuration.get(OrcConf.STRIPE_SIZE.getAttribute()));
```

Comment:
Changed this to using OrcConf to get value directly so no need to know property key detail.